### PR TITLE
Add LLM Inference sample for Web

### PR DESCRIPTION
**Pull request by Linchenn**

### CODE REVIEW

File: examples/llm_inference/js/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+<!doctype html>
+<html lang="en">
+<head>
+    <title>BYOM-Web Demo</title>
```
Comment (ktonthat) [inline examples/llm_inference/js/index.html:18]: Maybe call it "LLM Inference Web Demo"

Comment (Linchenn) [reply examples/llm_inference/js/index.html]: Nice catch, done!

File: examples/llm_inference/js/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# MediaPipe LLM Inference task for web
+
+## Overview
+
+This web sample demonstrates how to use the LLM Inference API to run common text-to-text generation tasks like information retrieval, email drafting, and document summarization, on web.
+
+
+## Prerequisites
+
+* A device that can access Chrome and has WebGPU support (Desktop and Laptop are recommanded).
```
Comment (schmidt-sebastian) [inline examples/llm_inference/js/README.md:10]: "A browser with WebGPU support (such as Chrome, Edge or Opera)"

File: examples/llm_inference/js/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+async function runDemo() {
+  const genaiFileset = await FilesetResolver.forGenAiTasks(
+      'https://cdn.jsdelivr.net/npm/@mediapipe/tasks-genai/wasm');
+  let llmInference;
+
+  submit.onclick = () => {
+    output.textContent = '';
+    submit.disabled = true;
+    llmInference.generateResponse(input.value, displayNewTokens).then(() => {
+      submit.disabled = false;
```
Comment (schmidt-sebastian) [inline examples/llm_inference/js/index.js:50]: Last comment - move this to the "if (complete)" block.

File: examples/llm_inference/js/index.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    });
+  };
+
+  submit.value = 'Loading the model...'
+  LlmInference
+      .createFromModelPath(
+          genaiFileset,
+          modelFileName,
+          )
```
Comment (schmidt-sebastian) [inline examples/llm_inference/js/index.js:59]: can this be one line?

File: examples/llm_inference/js/index.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      .then(llm => {
+        llmInference = llm;
+        submit.disabled = false;
+        submit.value = 'Get Response'
+      }).catch(() =>{
```
Comment (schmidt-sebastian) [inline examples/llm_inference/js/index.js:64]: let's remove. I want this be concise.

**Review by schmidt-sebastian — APPROVED** [review]

**Pull request merged**